DNN Platform/Website/Providers/DataProviders/SqlDataProvider/DotNetNuke.Data.SqlDataProvider

```diff
@@ -210,7 +210,12 @@ DECLARE @CreateOn datetime = {ts '2000-01-01 00:00:00'}
 DECLARE @UpdateOn datetime = NULL
 DECLARE @UpdateOnNotNull datetime = {ts '2000-01-01 00:00:00'}
 
-PRINT(N'Add 146 rows to {databaseOwner}[{objectQualifier}EventLogTypes]')
+PRINT(N'Add 150 rows to {databaseOwner}[{objectQualifier}EventLogTypes]')
+INSERT INTO {databaseOwner}[{objectQualifier}EventLogTypes] ([LogTypeKey], [LogTypeFriendlyName], [LogTypeDescription], [LogTypeOwner], [LogTypeCSSClass]) VALUES (N'APITOKEN_AUTHENTICATION_FAILED', N'API Token Authentication Failed', N'', N'DotNetNuke.Logging.EventLogType', N'OperationFailure')
+INSERT INTO {databaseOwner}[{objectQualifier}EventLogTypes] ([LogTypeKey], [LogTypeFriendlyName], [LogTypeDescription], [LogTypeOwner], [LogTypeCSSClass]) VALUES (N'APITOKEN_CREATED', N'API Token Created', N'', N'DotNetNuke.Logging.EventLogType', N'GeneralAdminOperation')
+INSERT INTO {databaseOwner}[{objectQualifier}EventLogTypes] ([LogTypeKey], [LogTypeFriendlyName], [LogTypeDescription], [LogTypeOwner], [LogTypeCSSClass]) VALUES (N'APITOKEN_DELETED', N'API Token Deleted', N'', N'DotNetNuke.Logging.EventLogType', N'GeneralAdminOperation')
+INSERT INTO {databaseOwner}[{objectQualifier}EventLogTypes] ([LogTypeKey], [LogTypeFriendlyName], [LogTypeDescription], [LogTypeOwner], [LogTypeCSSClass]) VALUES (N'APITOKEN_REVOKED', N'API Token Revoked', N'', N'DotNetNuke.Logging.EventLogType', N'GeneralAdminOperation')
+INSERT INTO {databaseOwner}[{objectQualifier}EventLogTypes] ([LogTypeKey], [LogTypeFriendlyName], [LogTypeDescription], [LogTypeOwner], [LogTypeCSSClass]) VALUES (N'APPLICATION_SHUTTING_DOWN', N'Application Shutting Down', N'', N'DotNetNuke.Logging.EventLogType', N'GeneralAdminOperation')

```

Comment:
Was this one supposed to be added? It doesn't appear to be related to API tokens…
```suggestion
```